atlascope/settings.py

```diff
@@ -18,6 +18,8 @@ class AtlascopeMixin(ConfigMixin):
     ROOT_URLCONF = 'atlascope.urls'
 
     BASE_DIR = Path(__file__).resolve(strict=True).parent.parent
+    DATASET_STORAGE_DIR = Path('data')
```

Comment:
```suggestion
    MEDIA_ROOT = BASE_DIR / Path('data')
```

Two things are changed here:
1. Uses [`MEDIA_ROOT`](https://docs.djangoproject.com/en/4.0/ref/settings/#std:setting-MEDIA_ROOT).
2. Ensuring it's referenced as an absolute path.